lib/components/CIDRField.dart

```diff
@@ -52,57 +53,55 @@ class _CIDRFieldState extends State<CIDRField> {
   Widget build(BuildContext context) {
     var textStyle = CupertinoTheme.of(context).textTheme.textStyle;
 
-    return Container(
-      child: Row(
-        children: <Widget>[
-          Expanded(
-            child: Padding(
-              padding: EdgeInsets.fromLTRB(6, 6, 2, 6),
-              child: IPField(
-                help: widget.ipHelp,
-                ipOnly: true,
-                textPadding: EdgeInsets.all(0),
-                textInputAction: TextInputAction.next,
-                textAlign: TextAlign.end,
-                focusNode: widget.focusNode,
-                nextFocusNode: bitsFocus,
-                onChanged: (val) {
-                  if (widget.onChanged == null) {
-                    return;
-                  }
-
-                  cidr.ip = val;
-                  widget.onChanged!(cidr);
-                },
-                controller: widget.ipController,
-              ),
-            ),
-          ),
-          Text("/"),
-          Container(
-            width: Utils.textSize("bits", textStyle).width + 12,
-            padding: EdgeInsets.fromLTRB(2, 6, 6, 6),
-            child: SpecialTextField(
-              keyboardType: TextInputType.number,
-              focusNode: bitsFocus,
-              nextFocusNode: widget.nextFocusNode,
-              controller: widget.bitsController,
+    return Row(
+      children: <Widget>[
+        Expanded(
+          child: Padding(
+            padding: EdgeInsets.fromLTRB(6, 6, 2, 6),
+            child: IPField(
+              help: widget.ipHelp,
+              ipOnly: true,
+              textPadding: EdgeInsets.all(0),
+              textInputAction: TextInputAction.next,
+              textAlign: TextAlign.end,
+              focusNode: widget.focusNode,
+              nextFocusNode: bitsFocus,
               onChanged: (val) {
                 if (widget.onChanged == null) {
                   return;
                 }
 
-                cidr.bits = int.tryParse(val) ?? 0;
+                cidr.ip = val;
```

Comment:
This diff is confusing, all that happened in this file is removing an empty wrapping `Container`, this text matches but is just visible lower in the diff